File: code/backend/webflux/v5/customer/webflux_customer_service/src/main/java/at/bumzack/foryouandyourfakewebshop/customerservice/model/Customer.java
package at.bumzack.foryouandyourfakewebshop.customerservice.model;

import org.springframework.data.relational.core.mapping.Table;

import java.time.LocalDateTime;

@Table("customer")
public record Customer(Long id, String firstName, String lastName, String email, String password, LocalDateTime created) {
}

File: code/backend/csharp/CustomerService/CustomerService/Customer.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CustomerService;


[Table("customer")]
public class Customer
{

    [Key, Required]
    public long Id { get; set; }

    public string? FirstName  { get; set; }

    public string? LastName { get; set; }
    
    public string Email { get; set; }

    public string Password { get; set; }

    public DateTime? Created { get; set; }
}
